src/assets/scripts/airport/AirportModel.js

```diff
@@ -95,8 +95,8 @@ export default class AirportModel {
         // TODO: why is this var getting reassigned to a magic number?
         s = 100;
         const speed_factor = sin((s + window.gameController.game_time()) * 0.5) + sin((s + window.gameController.game_time()) * 2);
-        wind.angle += crange(-1, angle_factor, 1, degreesToRadians(-4), degreesToRadians(4));
-        wind.speed *= crange(-1, speed_factor, 1, 0.9, 1.05);
+        wind.angle += extrapolate_range_clamp(-1, angle_factor, 1, degreesToRadians(-4), degreesToRadians(4));
+        wind.speed *= extrapolate_range_clamp(-1, speed_factor, 1, 0.9, 1.05);
```

Comment:
what do all these numbers mean?  In the future I'd like to see these enumerated. Math is hard and I, or anybody else that may want to contribute, may not be able to figure out what each number is for or why a specific value is used.